### ISSUE TITLE
Most controller overlays don't indicate d-pad status

### ISSUE THREAD

**Issue by keithbowes** [libretro/common-overlays#82]
When you push say A button or Select with most controller overlays, there's an indication that the button was pressed (e.g. the button becomes brighter), but that's not the case with most controller overlays, where the d-pad remains the same even when you push the d-pad buttons.  There are a few exceptions (but they have bad contrast between pressed and unpressed buttons in general), but the most usable ones are affected by this bug/deficit.

**Comment by hizzlekizzle** [comment]
yeah, the problem is that most of the dpads look very bad if you try to slice up the directions into separate png images, so they keep them as one big image and then use invisible hitboxes for the actual directions.

**Comment by DJM75** [comment]
> When you push say A button or Select with most controller overlays, there's an indication that the button was pressed (e.g. the button becomes brighter), but that's not the case with most controller overlays, where the d-pad remains the same even when you push the d-pad buttons. There are a few exceptions (but they have bad contrast between pressed and unpressed buttons in general), but the most usable ones are affected by this bug/deficit.

Hi, you can try my recently added RGPAD, it doesn't have this problem ;)

**Comment by DJM75** [comment]
> When you push say A button or Select with most controller overlays, there's an indication that the button was pressed (e.g. the button becomes brighter), but that's not the case with most controller overlays, where the d-pad remains the same even when you push the d-pad buttons. There are a few exceptions (but they have bad contrast between pressed and unpressed buttons in general), but the most usable ones are affected by this bug/deficit.

Hi, you can try my recently added RGPAD, it doesn't have this problem ;)

**Comment by DJM75** [comment]
> When you push say A button or Select with most controller overlays, there's an indication that the button was pressed (e.g. the button becomes brighter), but that's not the case with most controller overlays, where the d-pad remains the same even when you push the d-pad buttons. There are a few exceptions (but they have bad contrast between pressed and unpressed buttons in general), but the most usable ones are affected by this bug/deficit.

Hi, you can try my recently added RGPAD, it doesn't have this problem.
